lib/LANraragi/Utils/Routing.pm

```diff
@@ -158,6 +158,7 @@ sub apply_routes {
     $public_api->get('/api/tankoubons')->to('api-tankoubon#get_tankoubon_list');
     $public_api->get('/api/tankoubons/:id')->to('api-tankoubon#get_tankoubon');
     $logged_in_api->put('/api/tankoubons')->to('api-tankoubon#create_tankoubon');
+    $logged_in_api->put('/api/tankoubons/:id')->to('api-tankoubon#update_metadata');
```

Comment:
I think this API should be the definite endpoint for updating tankoubon data instead of PUT` /api/tanks/:id/archive` .  

It follows the convention of the /api/archive endpoints -- Not sure why I accepted the current one to begin with but since the feature isn't public yet, might as well fix it here.